samples/dashboards/demo/datacursor-sync/demo.js

```diff
@@ -10,8 +10,18 @@ Dashboards.board('container', {
             id: 'dashboards-layout-1',
             rows: [{
                 cells: [{
+                    responsive: {
+                        small: {
```

Comment:
Responsive options (going from small to large) don't seem to take effect unless the page is reloaded, is this a bug or config?

Reply:
I think this is a bug